R/data_maggie.R

```diff
@@ -0,0 +1,464 @@
+#'  \item{white_percent}{Percent of fans who are white}
+#'  \item{nonwhite_percent}{Percent of fans who are not white}
+#'  }  
+#'@source See \url{https://github.com/fivethirtyeight/data/tree/master/nfl-fandom/NFL_fandom_data-surveymonkey.csv}
+#' To convert data frame to tidy data (long) format, run:
+#' library(tidyverse)
+#' nfl_fandom_surveymonkey_tidy <- nfl_fandom_surveymonkey %>%
+#' gather(key= race_party, value=percent, 3:20)
+
+
+#'The Media Really Started Paying Attention to Puerto Rico When Trump Did
+#'
+#'The raw data behind the story
+#'"The Media Really Started Paying Attention to Puerto Rico When Trump Did"
+#'\url{https://fivethirtyeight.com/features/the-media-really-started-paying-attention-to-puerto-rico-when-trump-did/}
+#'
+#'Google Trends Data:
+#'@format A data frame with 37 rows representing dates and 5 variables:
+#'\describe{
+#'  \item{date}{Date}
+#'  \item{hurricane_harvey_us}{US Google search interest on the specified date for Hurricane Harvey}
+#'  \item{hurricane_irma_us}{US Google search interest on the specified date for Hurricane Irma}
+#'  \item{hurricane_maria_us}{US Google search interest on the specified date for Hurricane Maria}
+#'  \item{hurricane_jose_us}{US Google search interest on the specified date for Hurricane Jose}
+#'  }
+#'@details Google search interest is measured in search term popularity relative to peak popularity in the given 
+#'region and time period (with 100 as peak popularity)
+#'@source Google Trends \url{https://trends.google.com/trends/}
+#'  
+#'Mediacloud Hurricanes Data:
+#'@format A data frame with 38 rows representing dates and 5 variables:
+#'\describe{
+#'  \item{date}{Date}
+#'  \item{harvey}{The number of sentences in online news which mention Hurricane Harvey on the specified date}
+#'  \item{irma}{The number of sentences in online news which mention Hurricane Irma}
+#'  \item{maria}{The number of sentences in online news which mention Hurricane Maria}
+#'  \item{jose}{The number of sentences in online news which mention Hurricane Jose}
+#'  }
+#'@source Mediacloud \url{https://mediacloud.org/}
+#'  
+#'Mediacloud States Data:
+#'@format A data frame with 51 rows representing dates and 4 variables:
+#'\describe{
+#'  \item{date}{Date}
+#'  \item{texas}{The number of sentences in online news which mention Texas on the specified date}
+#'  \item{puerto_rico}{The number of sentences in online news which mention Puerto Rico}
+#'  \item{florida}{The number of sentences in online news which mention Florida}
+#'  }  
+#'@source Mediacloud \url{https://mediacloud.org/}
+#'  
+#'Mediacloud Top Online News Data: 
+#'@format A data frame with 49 rows representing media outlets and 2 variables:
+#'\describe{
+#'  \item{name}{Name of media outlet source included in Media Cloud's "U.S. Top Online News" collection}
+#'  \item{url}{URL of corresponding media outlet source}
+#'}
+#'@source Mediacloud \url{https://mediacloud.org/}
```

Comment:
Since `google_trends, mediacloud_hurricanes, mediacloud_states, etc` all come from the same article and 538/github folder: `data-raw/puerto-rico-media/`, it would be a good idea to cross-reference them in a `@seealso` tag, so that users can quickly jump between them. 

Run `?classic_rock_raw_data` and `?classic_rock_song_list` in the console and see the corresponding code in`R/data_albert.R` for a good example

Could you do the same for all datasets that are related as defined above?